### DISCUSSION TITLE
PatchTool 6.38 4/5 修正文本

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#310] [Announcements]
更新漢化至中國服6.25。適用6.38版本。技改更新至6.38。

檔案：

- FFXIVChnTextPatch-20230405.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230405.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230405.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.0.2.1'>PatchTool 6.38 4/5 修正文本</a>.</em>

**Comment by a61456711** [comment]
80等支線任務「悌陽象族的友人」，沒有提示要在說話頻道輸入日文或英文，嘗試輸入英文「Sushena」依然失敗，拿除漢化包輸入才能正常進入下一階段

**Reply by GpointChen** [reply]
沒有提示的話應該是我沒有改到，應該是要輸入中文。預計下次更新時一起修正。

**Comment by ksledu** [comment]
作者您好，想請教一下遊戲里的定型文的內容是在漢化檔那個文件里面？還是說不屬於文檔系統。  謝謝！
![tab](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/96337656/8abbc2bb-3900-4f5a-8dd5-b0b163414045)

**Reply by GpointChen** [reply]
Completion.csv

**Reply by ksledu** [reply]
谢谢
